lib/moab_showq_client.rb

```diff
@@ -17,24 +17,25 @@ def initialize(cluster)
   end
 
   def setup
-    scheduler = Moab::Scheduler.new(
-      host: @server['host'],
-      lib: @server['lib'],
-      bin: @server['bin'],
-      moabhomedir: @server['homedir']
-    )
+    lib= Pathname.new(@server['lib'].to_s)
+    bin= Pathname.new(@server['bin'].to_s)
+    moabhomedir= Pathname.new(@server['homedir'].to_s)
```

Comment:
Since `@server` is an instance var, it might be cleaner code to create separate methods `moab_lib`, `moab_bin`, and `moab_home` that all return pathnames.

Then lines 23-29 could be in a method `showq_summary_xml` and comments saying that it returns the equivalent of `showq -s --xml`.

Then the the setup method is just:

```ruby
def setup
  doc = REXML::Document.new(showq_summary_xml)
  self.active_jobs = doc.root.elements["queue[@option='active']"].attributes["count"].to_i
...
```